charts/rancher-k3s-upgrader/0.6.0/Chart.yaml

```diff
@@ -6,5 +6,5 @@ home: https://github.com/rancher/system-charts/blob/dev-v2.8/charts/rancher-k3s-
 sources:
   - "https://github.com/rancher/system-charts/blob/dev-v2.8/charts/rancher-k3s-upgrader"
 version: 0.6.0
-appVersion: v0.13.1
+appVersion: v0.13.4
```

Comment:
is this a mistake? I don't think we should mutate the `0.6.0` chart

Reply:
Good catch, split this into two commits and must have updated the wrong directory